lib/src/cardano_version.dart

```diff
@@ -13,13 +14,11 @@ class CardanoVersion with _$CardanoVersion {
     required int versionMinor,
     required int versionPatch,
     required bool locked,
+    required Flags_mode flags,
```

Comment:
You should not use the generated freezed class type unless there's a reason for it (accessing something that's otherwise not available). In your case (when there's a single factory/type) this is not needed. You should do `required Flags flags,`